ibis/expr/operations.py

```diff
@@ -720,6 +720,18 @@ class Quantile(Reduction):
     output_type = rules.type_of_arg(1)
 
 
+class MultiQuantile(Reduction):
+
+    input_type = [value,
+                  rules.collection(name='quantile'),
```

Comment:
`collection` here means "column or table". I think for now we want to be very strict about the inputs and we can expand support for different concrete inputs (like `Series`) as needed.